charts/tekton-apps/templates/eventlistener.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: triggers.tekton.dev/v1beta1
 kind: EventListener
 metadata:
-  name: {{ include "tekton-apps.eventlistenerName" (set "suffix" .Values.eventlistener.suffix }}
+  {{- template "tekton-apps.eventlistenerName" .Values.eventlistener.suffix -}}
```

Comment:
sorry :)
but template should bring the str value not key:value otherwise it become cryptic